samples/demos/sql-graph/recommendation-system/demo1-create-and-populate-nodes-and-edges.sql

```diff
@@ -0,0 +1,174 @@
+------------------------------------------------------------------------
+-- Event:        SQL Saturday #675 Parma, November 18 2017             -
+--               http://www.sqlsaturday.com/675/EventHome.aspx         -
+-- Session:      SQL Server 2017 Graph Database                        -
+-- Demo:         Demo1: Create and populate Nodes and Edges            -
+-- Author:       Sergio Govoni                                         -
+-- Notes:        --                                                    -
+------------------------------------------------------------------------
+
+USE [WideWorldImporters];
+GO
+
+------------------------------------------------------------------------
+-- Nodes                                                               -
+------------------------------------------------------------------------
+
+DROP TABLE IF EXISTS Nodes.Person;
+GO
+
+-- Create a node table
+CREATE TABLE Nodes.Person
+(
+  PersonID INTEGER NOT NULL PRIMARY KEY
+  ,FullName NVARCHAR(50) NOT NULL
+  ,[Language] NVARCHAR(50) NOT NULL
+) AS NODE;
+GO
+
+
+SELECT * FROM sys.tables WHERE is_node = 1;
+GO
+
+SELECT
+  *
+FROM
+  INFORMATION_SCHEMA.COLUMNS
+WHERE
+  Table_Schema = 'Nodes'
+  AND Table_Name = 'Person';
+GO
+
+
+/*
+SELECT FullName, CustomFields, * 
+FROM [Application].[People];
+GO
+*/
+
+INSERT INTO Nodes.Person
+(
+  PersonID
+  ,FullName
+  ,[Language]
+)
+SELECT
+  PersonID
+  ,FullName
+  ,[Language] = Languages.[Value]
+FROM
+  [Application].[People]
+CROSS APPLY
+  (SELECT * FROM OPENJSON (CustomFields, '$.OtherLanguages')) As Languages
```

Comment:
Are you just taking 0 element from the array? Is this equivalent to SELECT JSON_VALUE(CustomFields, '$[0]')?

Reply:
Hi Jovan,

Thanks for your suggestion, I used the function JSON_VALUE instead of OPENJSON.